_data/projects/wemake-python-styleguide.yml

```diff
@@ -0,0 +1,12 @@
+name: wemake-python-styleguide
+desc: The strictest and most opinionated python linter ever!
+site: https://github.com/wemake-services/wemake-python-styleguide/blob/master/CONTRIBUTING.md
+tags:
+- python
+- linter
+- flake8
+- code quality
+- oss
+upforgrabs:
+  name: "level:starter"
+  link: https://github.com/wemake-services/wemake-python-styleguide/issues?q=is%3Aissue+is%3Aopen+label%3Alevel%3Astarter
```

Comment:
@sobolevn Please change this to "https://github.com/wemake-services/wemake-python-styleguide/labels/level%3Astarter"